roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
+    containerized: "{{ containerized | default('no') | bool }}"
```

Comment:
It might be better to set containerized by default if running on atomic host, we do this with l_containerized in openshift_facts/tasks/main.yml